packages/openactive-integration-tests/CONTRIBUTING.md

```diff
@@ -0,0 +1,271 @@
+  testCategory: 'payment',
+  testFeature: 'simple-book-with-payment',
+  testFeatureImplemented: true,
+  testIdentifier: 'with-payment-property',
+```
+
+Will exist in the path `test/features/payment/simple-book-with-payment/implemented/with-payment-property-test.js`.
+
+### feature.json
+
+For each feature, there also exists a `feature.json` file, which contains metadata about the feature. This is used by the [Documentation Generator](./ARCHITECTURE.md#documentation-generator) to automatically generate the documentation for that feature. It exists at the following path:
+
+`test/features/<testCategory>/<testFeature>/feature.json`
+
+Here is an example `feature.json` file:
+
+```json
+{
+  "category": "payment",
+  "identifier": "non-free-opportunities",
+  "name": "Opportunities with a non-zero price",
+  "description": "The most simple form of booking with payment. Does not check for leases.",
+  "explainer": "",
+  "specificationReference": "https://www.openactive.io/open-booking-api/EditorsDraft/#step-by-step-process-description",
+  "required": false,
+  "coverageStatus": "complete",
+  "links": [
+    {
+      "name": ".NET Tutorial",
+      "href": "https://tutorials.openactive.io/open-booking-sdk/quick-start-guide/storebookingengine/day-5-b-and-delete-order"
+    }
+  ],
+  "requiresOneOfIfImplemented": ["prepayment-optional", "prepayment-required",  "prepayment-unavailable"]
+}
+```
+
+## Anatomy of a test
+
+Most tests are configured using `FeatureHelper.describeFeature(..)`. A simple example usage of this is:
+
+```js
+FeatureHelper.describeFeature(module, {
+  testCategory: 'core',
+  testFeature: 'availability-check',
+  testFeatureImplemented: true,
+  testIdentifier: 'availability-confirmed',
+  testName: 'Occupancy in C1 and C2 matches feed',
+  testDescription: 'Runs C1 and C2 for a known opportunity from the feed, and compares the results to those attained from the feed.',
+  testOpportunityCriteria: 'TestOpportunityBookable',
+  controlOpportunityCriteria: 'TestOpportunityBookable',
+},
+function (configuration, orderItemCriteriaList, featureIsImplemented, logger) {
+```
+
+An explanation of the main properties:
+
+* `testCategory`, `testFeature`, `testIdentifier`: Identifiers for the **Category**, **Feature** and **Test** respectively. For info on what these are, see [Structure](#structure).
+* `testFeatureImplemented`: Whether or not this test is **implemented** or **non-implemented**. For info on what this means, see [Structure](#structure).
+* `testName`, `testDescription`: Short- and long-form human-readable descriptions of the text. These are used for automatic Documentation Generation and for the test report.
+* `testOpportunityCriteria`: The **Opportunity Criteria** to use for the primary OrderItem under test. For info on criteria, see [Opportunity Criteria](#opportunity-criteria).
+* `controlOpportunityCriteria`: The **Opportunity Criteria** to use for any "control" OrderItems.
```

Comment:
```suggestion
* `controlOpportunityCriteria`: The **Opportunity Criteria** to use for any "control" OrderItems. For info on criteria, see [Opportunity Criteria](#opportunity-criteria).
```

Reply:
A dev might not necessarily read the line about `testOpportunityCriteria`

Reply:
Great point. Done